hosting/src/app/games/quiz/question/question.component.ts

```diff
@@ -67,4 +67,11 @@ export class QuestionComponent implements OnChanges {
             .subscribe();
     }
 
+    public getRoundProgress(): number {
+        return (<Array<string>> this.quiz.roundList).findIndex((roundUid) => this.round.uid === roundUid) + 1;
+    }
+
+    public getQuestionProgress(): number {
+        return (<Array<string>> this.round.questionList).findIndex((questionUid) => this.question.uid === questionUid) + 1;
```

Comment:
```suggestion
        return <Array<string>> this.round.questionList).indexOf(this.question.uid) + 1;
```

not sure the more complex find index is needed here?

Reply:
Good call 👍